cognite/client/_api_client.py

```diff
@@ -65,7 +65,7 @@
     from cognite.client import CogniteClient
     from cognite.client.config import ClientConfig
 
-log = logging.getLogger("cognite-sdk")
+log = logging.getLogger(__name__)
```

Comment:
We should make an announcement about this change in the sdk and python channels.